crates/turborepo-api-client/src/lib.rs

```diff
@@ -111,10 +120,16 @@ pub struct UserResponse {
     pub user: User,
 }
 
+pub struct PreflightResponse {
+    location: Url,
+    allow_auth: bool,
```

Comment:
I prefer longer variable names. 🤷‍♂️

```suggestion
    allow_authorization_header: bool,
```